### DISCUSSION TITLE
Agent downgrade 1.4.12

### DISCUSSION THREAD

**Discussion by thewalkerz** [amidaware/tacticalrmm#1442] [Q&A]
Hi tactical members,

I'm new using tactical, I have a problem with the tactical agent, when I update the agent to the latest version, after 1 hour or so the agent is outdated to version 1.4.12, it has happened to me with several computers and I can not find the reason why.

**Comment by wh1te909** [comment]
what version of tacticalrmm are you running? where are you seeing 1.4.12?
